wsdl/ver10/schema/onvif.xsd

```diff
@@ -7636,6 +7636,11 @@ and sample rate. </xs:documentation>
 					<xs:documentation>URI provided by the service supplying data to be recorded. A device shall support at least 128 characters.</xs:documentation>
 				</xs:annotation>
 			</xs:element>
+			<xs:element name="VideoSourceToken" type="tt:ReferenceToken" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>The video source token of all recordings that will use this configuration.</xs:documentation>
```

Comment:
1. VideoSourceToken parameter in RecordingSourceInformation makes the recording container only record the data of this particular video source.  As Per ONVIF Recording Control specification, the recording container can support recording two video tracks, one containing a low resolution or low frame rate stream and one containing a high resolution or high frame rate stream of the same video source or different video source. 
2.  RecordingJobSource have Source Token i.e Profile token tells about which media should be recorded to the recording container. If the recording container already associated with video source via RecordingSourceInformation and if the video source in recording container is  different from video source present in RecordingJobSource Profile token, it creates in-computability issue and the job cannot record the particular media profile data to the specific recording container.

Reply:
Can you explain how 1. can be done with the current specification, because to my knowledge, you need to specify a MediaProfile which can only have 1 VideoEncoderConfiguration assigned to it. So you wouldn't be able to record different resolutions or video sources using only 1 RecordingJob.

2. Yes, this would be a new validation that would have to be done by the device to keep things in sync
3. Why wouldn't it be possible to change the SourceInformation?

Reply:
1. we can change media profile assigned to recording job, where the new media profile points to different video encoder, video source from the previous media profile.
3. After changing recording source information in the recording container, recorded data with old source in the container will reflect with new recording source when loading the recording timeline, that is not expected behavior.